voidcrew/modules/overmap/code/controllers/subsystem/shuttle.dm

```diff
@@ -1,15 +1,49 @@
 /datum/controller/subsystem/shuttle/proc/create_ship(datum/map_template/shuttle/voidcrew/ship_template_to_spawn)
-	// load a ship map
+	RETURN_TYPE(/obj/structure/overmap/ship)
 
 	UNTIL(!shuttle_loading)
-	ship_template_to_spawn = new
+	var/obj/structure/overmap/ship/ship_to_spawn = new(SSovermap.get_unused_overmap_square(tries = INFINITY))
+	ship_template_to_spawn = new ship_template_to_spawn
+
 	shuttle_loading = TRUE
-	if (!load_template(ship_template_to_spawn))
-		stack_trace("Failed to load ship!")
-		shuttle_loading = FALSE
-		return
+	var/obj/docking_port/mobile/loaded = action_load(ship_template_to_spawn)
 	shuttle_loading = FALSE
 
-	var/obj/structure/overmap/ship/ship_to_spawn = new(SSovermap.get_unused_overmap_square(tries = INFINITY))
-	ship_to_spawn.shuttle = preview_shuttle // preview shuttle is set in `load_template()`
-	return preview_shuttle
+	if(!loaded)
+		qdel(ship_to_spawn)
+		return
+
+	ship_to_spawn.name = loaded.name
+	ship_to_spawn.source_template = ship_template_to_spawn
+	ship_to_spawn.shuttle = loaded
+	return ship_to_spawn
+
+/client/add_admin_verbs()
+	. = ..()
+	add_verb(src, list(
+		.proc/respawn_ship,
+		.proc/spawn_specific_ship,
+	))
+
+/client/remove_admin_verbs()
+	. = ..()
+	remove_verb(src, list(
+		.proc/respawn_ship,
+		.proc/spawn_specific_ship,
+	))
+
+/client/proc/respawn_ship()
+	set name = "Respawn Initial Ship"
+	set category = "Overmap"
+	SSovermap.initial_ship?.shuttle?.intoTheSunset()
```

Comment:
I think it would be better to check if there's an initial ship and if so, give the admin a confirmation message so they know they're about to delete the existing initial ship